controllers/FavoriteCommentController.js

```diff
@@ -0,0 +1,196 @@
+import { FavoriteComment, Comment } from '../database/models';
+/**
+ * @author Daniel
```

Comment:
Why are you stating the author twice? Please delete the comment from line 2 to line 6. You should always have an empty line after your imports, add an empty line on line 2.

Reply:
If I delete it is going to be an error because JSDoc comment is required. I just removed the author name.
<img width="762" alt="Screen Shot 2019-03-13 at 14 40 29" src="https://user-images.githubusercontent.com/37615728/54279619-3225d400-459e-11e9-9a99-3177c03c5267.png">